packages/webapp/pages/squads/discover/my.tsx

```diff
@@ -2,13 +2,21 @@ import React, { ReactElement } from 'react';
 import { NextSeo } from 'next-seo';
 import { useAuthContext } from '@dailydotdev/shared/src/contexts/AuthContext';
 import { SquadList } from '@dailydotdev/shared/src/components/cards/squad/SquadList';
+import { useRouter } from 'next/router';
+import { squadCategoriesPaths } from '@dailydotdev/shared/src/lib/constants';
 import { getLayout } from '../../../components/layouts/FeedLayout';
 import { mainFeedLayoutProps } from '../../../components/layouts/MainFeedPage';
 import { SquadDirectoryLayout } from '../../../../shared/src/components/squads/layout/SquadDirectoryLayout';
 import { defaultSeo } from '../../../next-seo';
 
 function MySquadsPage(): ReactElement {
   const { squads } = useAuthContext();
+  const router = useRouter();
+
+  if (squads?.length === 0) {
```

Comment:
though we can leave that [PR](https://github.com/dailydotdev/apps/pull/3615) of mine then and just do this

Reply:
Yeah, I can just rollback the last commit of this PR which involved that.

Reply:
This is just to ensure when the user is viewing this page, there is at least a Squad, otherwise, we redirect them since this is not available in the tabs when that is the case.